release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -640,6 +640,13 @@ This is an alternative to explicitly setting an ASN in the `spec.peerASN` field.
 With this release, you can configure a Remote Direct Memory Access (RDMA) Container Network Interface (CNI) on Single Root I/O Virtualization (SR-IOV) to enable high-performance, low-latency communication between containers.
 When you combine RDMA with SR-IOV, you provide a mechanism to expose hardware counters of Mellanox Ethernet devices to be used inside Data Plane Development Kit (DPDK) applications.
 
+[id="ocp-release-notes-networking-sr-iov-mlx-secure-boot_{context}"]
+==== Support configuring the SR-IOV Operator on a Secure-Boot-enabled environment for Mellanox cards
+
+With this release, you can configure the Single Root I/O Virtualization (SR-IOV) Operator when the system has secure boot enabled. The SR-IOV Operator is configured after you first manually configure the firmware for Mellanox devices. With secure boot enabled the resilience of your system is enhanced, and a crucial layer of defense for the overall security of your computer is provided.
```

Comment:
Is this correct?

```suggestion
==== Support configuring the SR-IOV Network Operator on a Secure-Boot-enabled environment for Mellanox cards

With this release, you can configure the Single Root I/O Virtualization (SR-IOV) Network Operator when the system has secure boot enabled. The SR-IOV Operator is configured after you first manually configure the firmware for Mellanox devices. With secure boot enabled the resilience of your system is enhanced, and a crucial layer of defense for the overall security of your computer is provided.
```

Reply:
Thanks @mburke5678 for you feedback. Yes, your suggestion is more accurate and I have applied it.